### PR TITLE
(DOCSP-13657) Update max time ms to 60000 ms, update UI screenshots

### PR DESCRIPTION
**Pull request by JuliaMongo**
The [build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=6054153c2723a9e448c2d9b1) is clean.
[Jira](https://jira.mongodb.org/browse/DOCSP-13657)
Staging links:
1. [Adjust MAX TIME MS](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-13657/query/maxtimems/)
2. [Troubleshooting topic about MAX TIME MS](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-13657/schema/#troubleshooting)

Changes: 
- Changed 5000 to 60000, updated both screenshots to show the correct name of the field. Reusing the same image now in both places, for consistency.
- Updated the image that was in the second topic, just in case it is used somewhere else in the guide (OK if not).

Also, a question to @jeff-allen-mongo: I think that this [DOCSP-14294](https://jira.mongodb.org/browse/DOCSP-14294)  is a duplicate of the one being fixed by this PR. Could you please confirm and I will then close it? Thank you.

### CODE REVIEW

File: source/query/maxtimems.txt [hunk 1]
```diff
@@ -13,21 +13,21 @@ interrupts the operation.
    :figwidth: 720px
    :alt: MaxTimeMS Option
 
-The default :guilabel:`MaxTimeMS` value is ``5000``. Consider raising
-this value if you have a very large collection, or your operations
-are frequently timing out. You can also consider creating
+The default :guilabel:`MAX TIME MS` value is ``60000``, or 30 seconds.
```
Comment (mmarcon) [inline source/query/maxtimems.txt:16]:
```suggestion
The default :guilabel:`MAX TIME MS` value is ``60000``, or 60 seconds.
```

File: source/schema.txt [hunk 2]
```diff
@@ -357,19 +357,19 @@ collection you are analyzing is very large, causing MongoDB to stop the
...
-   .. figure:: /images/compass/schema-query-options.png
+   .. figure:: /images/compass/max-time-ms.png
       :alt: The Options button is on the right side of the query bar, 
-            next to the green Analyze button.
+            next to the Analyze button.
 
-2. Increase the value of ``maxTimeMS`` to accommodate your collection. 
-   ``maxTimeMS`` defaults to 5000 milliseconds, or 5 seconds, but large 
+2. Increase the value of :guilabel:`MAX TIME MS` to accommodate your collection.
+   :guilabel:`MAX TIME MS` defaults to 60000 milliseconds, or 30 seconds, but large
```
Comment (mmarcon) [inline source/schema.txt:369]:
```suggestion
   :guilabel:`MAX TIME MS` defaults to 60000 milliseconds, or 60 seconds, but large 
```

**Review by mmarcon — CHANGES_REQUESTED** [review]
Aside from those 2 adjustments, LGTM.

File: source/query/maxtimems.txt (hunk 1 above)
Comment (JuliaMongo) [reply source/query/maxtimems.txt:16]: of course!

File: source/schema.txt (hunk 2 above)
Comment (JuliaMongo) [reply source/schema.txt]: Addressed. What was I thinking?

**Review by mmarcon — APPROVED** [review]
LGTM!

**Comment by jeff-allen-mongo** [comment]
LGTM 🚀

**Review by jeff-allen-mongo — APPROVED** [review]

**Pull request merged**